.bazelrc

```diff
@@ -5,7 +5,7 @@ common --enable_bzlmod
 build --spawn_strategy=standalone
 
 # Setup Xcode configuration.
-build --xcode_version_config=//:host_xcodes
+build --xcode_version=15.4.0.15F31d
```

Comment:
maybe override this just for `build:explicit_modules`?

Reply:
also i still cannot build as is with xcode 16, i wonder if we just add this to line 418 of `rules/xcode_sdk_frameworks.bzl` to indicate that we for now just support xcode 15.4
```
        if not xcode_version_name.startswith("version15_4"):
            continue
```